### ISSUE TITLE
[Config] Per-mob Subspecies Config

### ISSUE THREAD

**Issue by Claycorp** [Lycanite/LycanitesMobs#104]
It would be much better if you can control what subspecies spawn instead of all on/off. EX: Gorgomites are annoying as is but when they have 3x health they turn into near impossible to kill unless you have decent armor and a weapon because they spawn so frequently and if they also spawn in blocks and that causes more to spawn. They always spawn subspecies of the same type on top of that.

The lack of control over the spawning of "Greater subspecies" is a problem. Lunar grues and Umber lobbers spawn way too often and make things far too dark in hardcore darkness.
The geonotchs are fine because they take a trigger to spawn but the others just spawn freely in places you can't get to so they start to stack up and you could have multiple of them.
Also "Greater subspecies" should have an extra drop controls. Along with regular drops.

TL:DR
- An option to disable per mob subspecies instead of all or none.
- Greater subspecies need to be treated more like its own mob with extra drops, spawning options and an option to disable the darkening when they spawn.

**Comment by Lycanite** [comment]
I'll look into adding a bunch of new config values for each mob.

**Comment by Lycanite** [comment]
Added in 1.16.2.2!
